packages/core/data-transfer/src/utils/json.ts

```diff
@@ -2,6 +2,14 @@ import { isArray, isObject, zip, isEqual, uniq } from 'lodash/fp';
 
 const createContext = (): Context => ({ path: [] });
 
+const isLooselyEqual = (a: unknown, b: unknown) => {
+  // eslint-disable-next-line eqeqeq
+  if (a == b) {
```

Comment:
`return a == b`?

Reply:
Yeah, that would work 😆

Originally I had some extra checks for different types but in the end this seemed to work best. TBH we don't really even need an "isLooselyEqual" at all, but it seemed somehow cleaner this way in case we need to add those specific checks in the future.